front/lib/resources/space_resource.ts

```diff
@@ -319,6 +319,19 @@ export class SpaceResource extends BaseResource<SpaceModel> {
       },
       transaction,
     });
+    await Promise.all(
```

Comment:
concurrentExecutor ideally even if we know there will be just one here for now.

Probably want to leave a comment that explains the situation and the motivation for deleting for when we separate the management of groups and spaces